extensions/ui/org.eclipse.smarthome.ui.paper/web-src/css/components.css

```diff
@@ -611,7 +611,10 @@ md-progress-circular.md-default-theme .md-inner .md-left .md-half-circle
 }
 
 .parameter>div {
```

Comment:
yes I know. but from what I tested in the browser the `.parameter>div` is obsolete. Can you check? maybe I missed something.